parcel/parcel.jl

```diff
@@ -87,64 +94,129 @@ function parcel_model(dY, Y, p, t)
-        C_i = r_i
-        dqi_dt_depo = 4 * π / ρ_air * (S_i - 1) * G_i * r_i * N_ice
+        if "Monodisperse" in droplet_size_distribution
+            # Deposition on existing crystals (assuming all are the same...)
+            r_i = cbrt(q_ice / N_ice / (4 / 3 * π) / ρ_ice * ρ_air)
+            C_i = r_i
+            dqi_dt_depo = 4 * π / ρ_air * (s_i - 1) * G_i * r_i * N_ice
+        elseif "Gamma" in droplet_size_distribution
+        # Condensation on existing droplets assuming n(r) = A r exp(-λr)
+            λ = cbrt(32 * π * N_ice / q_ice * ρ_ice / ρ_air)
+            #A = N_ice* λ^2
+            r_i = 2 / λ
+            C_i = r_i
+            dqi_dt_depo = 4 * π / ρ_air * (s_i - 1) * G_i * r_i * N_ice
+        elseif "Lognormal" in droplet_size_distribution || "Jensen_Example" in droplet_size_distribution
+            r_i = cbrt(q_ice / N_ice / (4 / 3 * π) / ρ_ice * ρ_air) * exp(σ)
+            dqi_dt_depo = 4 * π / ρ_air * (s_i - 1) * G_i * r_i * N_ice
+        end
         # TODO - check if r_i is non-zero if initial conditions say q_ice = 0
     end
 
     dql_dt_cond = FT(0)
     if "Condensation" in growth_modes && N_liq > 0
+        G_l = CMO.G_func(aps, tps, T, TD.Liquid())
         if "Monodisperse" in droplet_size_distribution
         # Condensation on existing droplets assuming all are the same
-            G_l = CMO.G_func(aps, tps, T, TD.Liquid())
             r_l = cbrt(q_liq / N_liq / (4 / 3 * π) / ρ_liq * ρ_air)
-            dql_dt_cond = 4 * π / ρ_air * (S_liq - 1) * G_l * r_l * N_liq
         elseif "Gamma" in droplet_size_distribution
         # Condensation on existing droplets assuming n(r) = A r exp(-λr)
-            G_l = CMO.G_func(aps, tps, T, TD.Liquid())
             λ = cbrt(32 * π * N_liq / q_liq * ρ_liq / ρ_air)
             #A = N_liq* λ^2
             r_l = 2 / λ
-            dql_dt_cond = 4 * π / ρ_air * (S_liq - 1) * G_l * r_l * N_liq
         end
+        dql_dt_cond = 4 * π / ρ_air * (s_liq - 1) * G_l * r_l * N_liq
+    end
+
+    dq_liq_dt_lv = dql_dt_cond   # from liq-vap transitions
+    if "Jensen_Example" in droplet_size_distribution
+        # from liq-ice transitions
+        dq_liq_dt_li = -dqi_dt_new_immers
+         # from ice-liq transitions
+        dq_ice_dt_il = -dq_liq_dt_li
+        # from vap-ice transitions
+        dq_ice_dt_iv = dqi_dt_new_depo + dqi_dt_depo + dqi_dt_new_hom
+    else
+        # from liq-ice transitions
+        dq_liq_dt_li = -dqi_dt_new_immers - dqi_dt_new_hom
+         # from ice-liq transitions
+        dq_ice_dt_il = -dq_liq_dt_li
+        # from vap-ice transitions
+        dq_ice_dt_iv = dqi_dt_new_depo + dqi_dt_depo
     end
 
     # Update the tendecies
-    dq_ice_dt = dqi_dt_new_depo + dqi_dt_depo + dqi_dt_new_immers
-    dq_liq_dt = dql_dt_cond - dqi_dt_new_immers
-    dS_l_dt = a1 * w * S_liq - (a2 + a3) * S_liq * dq_liq_dt - (a2 + a4) * S_liq * dq_ice_dt
+    dq_vap_dt = -dq_ice_dt_iv - dq_liq_dt_lv
```

Comment:
all these variables will have better names with the other PR so we know which direction the phase change is going